src/modules/item/export.ts

```diff
@@ -118,6 +126,35 @@ function buildItemEntityMetadata(collection: Collection, item: Item): CatalystIt
   }
 }
 
+function buildTPItemEntityMetadata(
+  collection: Collection,
+  item: Item,
+  itemHash: string,
+  tree: MerkleDistributorInfo
+): CatalystItem | CatalystTPItem {
+  const baseEntityData = {
+    id: buildThirdPartyURN(extractThirdPartyId(collection.urn), collection.id, item.id),
```

Comment:
Careful here, the second param is the name, and the third the token id. You can decode the URN and use the data from there

```ts
const decodedUrn = decodeUrn(collection.urn)
buildThirdPartyURN(
  decodedCollectionUrn.thirdPartyName,
  decodedCollectionUrn.thirdPartyCollectionId,
  decodedUrn.thirdPartyTokenId
)
```

Reply:
I updated the code to the following:
```
const decodedURN = decodeURN(collection.urn)
  if (decodedURN.type !== URNType.COLLECTIONS_THIRDPARTY) {
    throw new Error('URN is not a third party URN')
  }
  const { thirdPartyCollectionId, thirdPartyName } = decodedURN
  const baseEntityData = {
    id: buildThirdPartyURN(thirdPartyName, thirdPartyCollectionId!, item.id),
    ...getBaseEntityMetadata(item)
  }
```
But I have some doubts about it...
The first thing I'm not fully bought into, is the boilerplate needed so TypeScript infers the URN correctly and so I can destructure the `thirdPartyCollectionId` and `thirdPartyName` (the `if` does the trick).

And while I was writing the code, I feel that we're decoding a URN and then building it again with the same parameters but the tokenId, which is the last one... so I believe all that process could be replaced by:

```
id: `${collection.urn}:${item.id}`
```

Maybe it can be abstracted in a method like `appendTokenId` so the logic remains in the `urn` utils file in case we change the implementation, but I Think this is more straightforward.

What do you think @nicosantangelo ?